stonesoup/types/angle.py

```diff
@@ -194,6 +194,9 @@ def average(cls, angles, weights=None):
 
         return cls(result)
 
+    def to_plotly_json(self):
+        return float(self)
+
```

Comment:
Is this used?

Reply:
Yes, I believe it is needed to plot angles in using Plotly

Reply:
Would it be possible to use `float()` or `.dtype(np.float_)` in plotting calls instead?

Reply:
You could but you would have to repeat that code in every `plot_###` function in the `Plottery` and `PolarPlotterly` classes. Plotting `Angle` types using Plotly fails on my machine without it. To clarify, the function isn't called directly anywhere in stone-soup but it is called by the plotly package.

Reply:
The `default` function in plotly/_plotly_utils/utils.py tries to convert `Angle` types but fails currently. We could try and edit the `Angle` class so it is picked up by one of the other encode methods but this would probably raise other implications.

[Link to default function code](https://github.com/plotly/plotly.py/blob/57e4d1d33c67c5cc715bec5c3c240dd6f4c3b10d/packages/python/plotly/_plotly_utils/utils.py#L85 )

Reply:
Ah, I see. I'll merge now.

Thanks for link to the code. Looks like Plotly does have encode for Decimal type, but not for Real or other numbers, which would be useful to add. I may raise an issue with them.